tensorflow/core/kernels/BUILD

```diff
@@ -6378,6 +6378,14 @@ tf_cc_test_mkl(
         "//tensorflow/core:testlib",
     ]
 )
+tf_mkl_kernel_library(
```

Comment:
This change has already been pushed into the repository. I'm not sure why it's appearing here.